style.css

```diff
@@ -635,300 +622,231 @@ input {
     line-height: 48px;
   }
 
-  .main__shop a {
+  .shop__link {
     position: absolute;
     color: white;
     top: 310px;
     left: 45px;
     font-size: 20px;
     line-height: 19px;
-    /*width: 100%;*/
   }
 
-  .main__shop a::after {
+  .shop__link::after {
     content: "\2192";
-    display: inline-block; 
+    display: inline-block;
     font-size: 20px;
     margin-left: 5px;
     width: 20px;
   }
 
-  .main h3{
+  .main h3 {
     font-size: 30px;
     line-height: 29px;
-    letter-spacing: 0.5px; 
+    letter-spacing: 0.5px;
     margin-bottom: 20px;
   }
-  
-  .main__product {
+
+  .product {
     margin-bottom: 80px;
   }
-  
-  .main__product_item:nth-child(n+4) {
+
+  .product__content:nth-child(n+4) {
     display: none;
     max-width: none;
-  } 
-  
-  .main__product_line {
+  }
+
+  .product__line {
     display: none;
     max-width: none;
   }
   
-  .main__product_item {
+  .product__content {
     flex-basis: 300px;
   }
-  
-  .main__product_item img {
+
+  .product__img {
     width: 80%;
     display: inline-block;
   }
 
-  .main__product_name {
+  .product__name {
     font-size: 20px;
-    line-height: 19px;  
+    line-height: 19px;
   }
 
-  .main__product_price {
-    font: bolder 25px/24px montserratregular;
+  .product__price {
+    font: bolder 25px/24px Gotham Pro, sans-serif;
   }
-
-  .main__product_silver-circle {
+	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484031)